.github/workflows/build.yml

```diff
@@ -3,14 +3,15 @@ on: [pull_request]
 
 jobs:
   build:
+    name: Packages
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - name: Setup Node.js and Cache
         uses: ./.github/actions/nodejs
 
-      - name: Build demo
-        run: npm run build
```

Comment:
Demo's build is already checking here

https://github.com/Tinkoff/maskito/blob/c22739e31d19fc75ddd220814618572209ccb0a4/.github/workflows/e2e.yml#L16-L17